docs/src/ch/cahn-hilliard.md

```diff
@@ -0,0 +1,123 @@
+# The Cahn-Hilliard Equation
+
+```@setup INFO
+include(joinpath(Base.@__DIR__, "..", "..", "docinfo.jl"))
+info = DocInfo.Info()
+```
+
+For this example, Decapodes will model the Cahn-Hilliard equation. This equation describes the evolution of a binary fluid as its two phases separate out into distinct domains. Below is a high resolution preview of this model. Notice how the fluid has separated into distinct regions (blue and red) as well as the presence of a transition region.
+
+!["Cahn Hilliard sample"](CahnHilliard_Final.jpg)
+
+## Formulating the Equation
+
+We first load in our dependencies
+
+```@example DEC
+# AlgebraicJulia Dependencies
+using Catlab
+using CombinatorialSpaces
+using Decapodes
+using DiagrammaticEquations
+
+# External Dependencies
+using CairoMakie
+using ComponentArrays
+using GeometryBasics
+using LinearAlgebra
+using MLStyle
+using OrdinaryDiffEq
+using Random
+Point3D = Point3{Float64};
+nothing #hide
+```
+
+and then proceed to describe our physics using Decapodes.
+
+```@example DEC
+CahnHilliard = @decapode begin
+    C::Form0
+    (D, γ)::Constant
+    ∂ₜ(C) == D * Δ(C.^3 - C - γ * Δ(C))
+end
+
+to_graphviz(CahnHilliard)
+```
+
+In this equation `C` will represent the concentration of the binary fluid, ranging from `-1` to `1` to differentiate between different phases. We also have a diffusion constant `D` and a constant `γ` whose square root is the length of the transition regions. This formulation of the Cahn-Hilliard equation was drawn from the Wikipedia page on the topic found [here](https://en.wikipedia.org/wiki/Cahn%E2%80%93Hilliard_equation).
+
+## Loading the Data
+
+We now generate the mesh information. We'll run the equation on a triangulated grid. I hide the mesh visualization code for clarity.
```

Comment:
I -> we

Reply:
Why, every other page uses we and not I.

Reply:
See: https://github.com/AlgebraicJulia/Decapodes.jl/blob/7a9846bcb722c8c4150612e31e6b500a4a3469f2/docs/src/overview.md?plain=1#L105-L107

Reply:
I know. The last sentence here says I

Reply:
Oh, I misunderstood. Fixed.